src/pages/library/DataBrowserDetails.js

```diff
@@ -196,8 +197,9 @@ export const SidebarComponent = ({ dataObj, id }) => {
         ])
       ]),
       h(ButtonPrimary, {
-        disabled: dataObj.access !== datasetAccessTypes.GRANTED,
-        tooltip: dataObj.access === datasetAccessTypes.GRANTED ? '' : uiMessaging.controlledFeature_tooltip,
+        disabled: dataObj.access !== datasetAccessTypes.GRANTED || true,
+        tooltip: 'We are currently working on the link to workspace feature which will be available soon',
```

Comment:
I believe the UX guideline is to end sentences with a period (`.`).
```suggestion
        tooltip: 'We are currently working on the link to workspace feature which will be available soon.',
```